cg-spec/editors_draft.html

```diff
@@ -1420,6 +1420,112 @@ <h2>isTRIPLE</h2>
+
+        <p>Notice that the BNODE function has to be used to create a blank node; simply writing a blank node directly in an expression <a data-cite="SPARQL11-QUERY#rExpression">is not permitted</a>.</p>
+
+        <p>Notice also that the given example query is special in the following two ways: its GRAPH clause does not actually consider the content of the named graphs and the expressions in the BIND clauses do not contain variables. As another example, the GRAPH clause and the BIND clauses in the following query do.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+          PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+          PREFIX rdfg: <http://www.w3.org/2004/03/trix/rdfg-1/>
+
+          SELECT ( COUNT(?t1) AS ?t1Count )
+                 ( COUNT(?t2) AS ?t2Count )
+          WHERE {
+            GRAPH ?g {
+              ?g rdf:type rdfg:Graph .
+              BIND( TRIPLE(?g, rdf:type, rdfs:Resource) AS ?t1 )
+              BIND( TRIPLE(?x, rdf:type, rdfs:Resource) AS ?t2 )
+            }
+          }
+          -->
+        </pre>
+
+        <p>For every RDF-star dataset with |n|>0 named graphs, the result of this query consists of a single SPARQL-star solution mapping that is defined for the three variables introduced in the SELECT clause. The value of <code>?t1Count</code> is the number of named graphs in the queried dataset that contain a triple of the form (|u|, <code>rdf:type</code>, <code>rdfg:Graph</code>) such that |u| is the name of the named graph in the dataset. The value of <code>?t2Count</code> is always&nbsp;0&nbsp;(zero). The reason for the latter is that evaluating the expression of the second BIND clause results in an error because the variable&nbsp;<code>?x</code> is not bound in the scope of this evaluation. As a side note, the same would also be the case for the first BIND clause if the variable&nbsp;<code>?g</code> was not mentioned inside the GRAPH clause (e.g., if the given triple pattern was not there).</p>
```

Comment:
```suggestion
        <p>For every RDF-star dataset with |n|>0 named graphs, the result of this query consists of a single SPARQL-star solution mapping that is defined for the three variables introduced in the SELECT clause. The value of <code>?t1Count</code> is the number of named graphs in the queried dataset that contain a triple of the form (|u|, <code>rdf:type</code>, <code>rdfg:Graph</code>) such that |u| is the name of the named graph in the dataset. The value of <code>?t2Count</code> is always&nbsp;0&nbsp;(zero), because evaluating the expression of the second BIND clause results in an error, because the variable&nbsp;<code>?x</code> is not bound in the scope of this evaluation. As a side note, evaluating the expression of the first BIND clause would also result in an error, if the variable&nbsp;<code>?g</code> was not mentioned inside the GRAPH clause (e.g., if the given triple pattern was not there).</p>
```